module-bsp/board/rt1051/drivers/RT1051DriverI2C.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "RT1051DriverI2C.hpp"
 #include <log/log.hpp>
-#include "../board.h"
+#include <board.h>
```

Comment:
```suggestion
#include "board.h"
```
I think it is sufficient as `board.h` is the part of the same module/source directory `module-bsp`

Reply:
You're right :)